src/App.js

```diff
@@ -1,10 +1,32 @@
 import React from 'react';
-import './App.css';
+import withModal from './hoc/withModal';
+import Header from './components/Header/index';
+import withStoriesContainer from './hoc/withStoriesContainer';
+import { HashRouter as Router, Route } from 'react-router-dom';
+import FullStoryView from './components/Stories/FullStoryView';
+import StoriesContainer from './components/Stories/StoriesContainer';
 
-class App extends React.Component {
-  render() {
-    return null;
-  }
+import './App.scss';
+import './style/base.scss';
+import './style/reset.scss';
+import './style/style.scss';
+
+let EnhancedStoryView = withModal(FullStoryView);
```

Comment:
If we need to always add HOC for this component then we can add it while exporting. 
eg: export default withModal(FullStoryView) on FullStoryView.js file.